src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -655,4 +655,24 @@ public String getUpdatedByUserId() {
   public DateTime getOriginalDueDate() {
     return originalDueDate;
   }
+
+  public ItemStatus getItemStatus() {
+    if (item != null) {
+      return item.getStatus();
+    }
+
+   return ItemStatus.from(getItemStatusName());
+  }
+
+  public String getItemStatusName() {
+    if (item != null) {
+      return item.getStatusName();
+    }
+
+    if (representation.containsKey(LoanProperties.ITEM_STATUS)) {
+      return representation.getString(LoanProperties.ITEM_STATUS);
+    }
+
+    return null;
```

Comment:
Could this instead use the JsonPropertyFetcher, as that would return null if the property has no value?